Add instructions for viewing build-time OpenAPI logs (#36071)
dotnet/AspNetCore.Docs: aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

To add support for generating OpenAPI documents at build time, install the `Microsoft.Extensions.ApiDescription.Server` package:

> ## View build-time OpenAPI logs (Terminal Logger)

Reviewer: This should be moved furhter down after the instructions about installation.

> When `Microsoft.Extensions.ApiDescription.Server` runs the **:::no-loc text="GetDocument":::** step during `dotnet build`, progress messages may not appear with the .NET Terminal Logger at default verbosity in .NET 8 or later. To surface these messages while building, use either of the following options with the `dotnet build` command.

Reviewer: ```suggestion
When `Microsoft.Extensions.ApiDescription.Server` runs the **:::no-loc text="GetDocument":::** step during `dotnet build`, progress messages are not visible with the .NET Terminal Logger at default verbosity in .NET 8 or later. To surface these messages while building, use either of the following options with the `dotnet build` command.
```
